patches/solc.diff

```diff
@@ -228,21 +228,7 @@ index 517f49c35331..e2dc892108fb 100644
  	bool useStaticCall = funKind == FunctionType::Kind::BareStaticCall || (_functionType.stateMutability() <= StateMutability::View && m_context.evmVersion().hasStaticCall());
 
  	if (_tryCall)
-@@ -2762,7 +2765,12 @@ void ExpressionCompiler::appendExternalFunctionCall(
- 		m_context << Instruction::DUP2;
- 	}
-
--	// CALL arguments: outSize, outOff, inSize, inOff (already present up to here)
-+	// Append `0` to the stack for `AUTHCALL`'s `valueExt`.
-+	// The EIP-3074 spec says that the `valueExt` parameter is always supposed to be `0`.
-+	if (isAuthCall)
-+		m_context << u256(0);
-+
-+	// CALL arguments: outSize, outOff, inSize, inOff, [valueExt (if AUTHCALL)] (already present up to here)
- 	// [value,] addr, gas (stack top)
- 	if (isDelegateCall)
- 		solAssert(!_functionType.valueSet(), "Value set for delegatecall");
```

Comment:
This is the first time I work with .diff files - how exactly does it work if a line is missing from the diff files when applying the patches? Here I would've assumed that this line shouldn't be removed but perhaps it has no effect

Reply:
makes perfect sense - thank you for the explanation!